soc/espressif/esp32c3/Kconfig.soc

```diff
@@ -19,6 +19,12 @@ config SOC_ESP32C3_MINI_N4
 	help
 	  ESP32C3_MINI_N4
 
+config SOC_ESP32C3_ORTHOSIE1
```

Comment:
Please, remove this entry and use `SOC_ESP32C3` in the board definition.

Reply:
Thank you, yes, that makes sense. I will push the updated version.